HW1/P4/P4_bfs.py

```diff
@@ -0,0 +1,35 @@
+def filterer(i):
+    def filt(x):
+        return x==itr_acc.value
+    return filt
```

Comment:
I'm a little confused by this. Why do you need this nested function structure? You're basically just doing a filter to only add nodes from the present iteration right? Can't you just do that with a lambda function?